.ci/releasenotes-gen-xdoc-push.sh

```diff
@@ -0,0 +1,47 @@
+#!/bin/bash
+set -e
+
+source ./.ci/util.sh
+
+checkout_from https://github.com/checkstyle/contribution
+
+cd .ci-temp/contribution/releasenotes-builder
+mvn -e --no-transfer-progress clean compile package
+cd ../../../
+
+if [ -d .ci-temp/checkstyle ]; then
+  cd .ci-temp/checkstyle/
+  git reset --hard origin/master
+  git pull origin master
+  git fetch --tags
+  cd ../../
+else
+  cd .ci-temp/
+  git clone https://github.com/checkstyle/checkstyle
+  cd ../
+fi
+
+CS_RELEASE_VERSION="$(getCheckstylePomVersion)"
+echo CS_RELEASE_VERSION="$CS_RELEASE_VERSION"
+
+cd .ci-temp/checkstyle
+LATEST_RELEASE_TAG=$(git describe --tags --abbrev=0)
+echo LATEST_RELEASE_TAG="$LATEST_RELEASE_TAG"
+
+cd ../
+
+BUILDER_TEMPLATE_DIR="contribution/releasenotes-builder/src/main/resources/com/github/checkstyle/\
+/templates/"
```

Comment:
Please try to play with name or introduce new variable that store path to repo.

Wrapping string content is very error prone approach, we need to avoid it